### ISSUE TITLE
Display of in-statement properties needs to differentiate 1st-level and 2nd-level properties

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#1563]
Display of in-statement properties in Detail needs to differentiate 1st-level and 2nd-level properties. 2nd-level property is a property of the property - thus, even only labelling it differently is not a good solution; better would be to bind those 2nd-level properties in the display with the 1st-level properties, gray the 1st level out somehow to signal that it is not in it that this entity appears, and indent the second level underneath those 1st-level properties.

![obrazek](https://user-images.githubusercontent.com/55400794/214554524-4eb866a9-0b2c-4fc4-b170-2fa77b4f67b9.png)

**Comment by Ptrhnk** [comment]
@davidzbiral 

<img width="531" height="436" alt="Image" src="https://github.com/user-attachments/assets/9a6e5c42-0348-4a71-a732-ec104d0190c3" />

**Comment by davidzbiral** [comment]
@Ptrhnk Yes, on principle - I do not know what differentiates your mock data row 2 and 3, is it third-level properties? You are right that we also need 3rd-level props displayed differently from 2nd-level props, if this is what you are showing with the mockup.
